kubernetes/ingress/supernovahunters.org.yaml

```
@@ -0,0 +1,46 @@
+apiVersion: cert-manager.io/v1
+kind: Certificate
+metadata:
+  name: supernovahunters-org-tls
+  labels:
+    use-azuredns-solver: "true"
+spec:
+  issuerRef:
+    name: letsencrypt-prod
+    kind: ClusterIssuer
+  secretName: supernovahunters-org-tls
+  dnsNames:
+    - supernovahunters.org
+    - "*.supernovahunters.org"
```

do we need these wildcard subdomains? If not you can explicitly list the subdomains, e.g. `www.supernovahunters.org` and avoid the dns solver, it can use the HTTP solver instead (quicker)

i.e. doesn't need this label
```
  labels:
    use-azuredns-solver: "true"
```

 Noting our nginx server blocks don't catch the wildcard subdomains so we should switch to explicit domains here and remove the `use-azuredns-solver` label
```suggestion
    - "www.supernovahunters.org"
```